commcare_connect/opportunity/helpers.py

```diff
@@ -280,18 +281,22 @@ def get_opportunity_list_data(organization, program_manager=False):
     today = now().date()
     three_days_ago = now() - timedelta(days=3)
 
+    pending_approvals_sq = Subquery(
+        UserVisit.objects.filter(opportunity_access__opportunity_id=OuterRef("pk"), status="pending")
+        .values("opportunity_access__opportunity_id")
```

Comment:
why is the value the opportunity id, rather than the visit id? won't this always count one if we count distinct  values in the annotation (since they all have the same opportunity id)

Reply:
Since this comes before the annotate, the opp-id is only used for grouping by. And the following `count=Count("id", distinct=True)` counts distinct visit IDs, grouped by opp-id.